atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -73,6 +73,10 @@ def run(self):
         koji_config = get_koji(self.workflow, {})
         insecure = koji_config.get('insecure_download', False)
         urls = self.get_srpm_urls(signing_intent['keys'], insecure=insecure)
+
+        if not urls:
+            raise RuntimeError('No srpms found for source container')
```

Comment:
Should we inform the build is being aborted because this would generate an empty source container image? Could either be in the Error message or just a comment to be explicit on why we are aborting here.

Reply:
added error  msg

Reply:
IMHO what athos meant was reporting to user via exception that source container image is empty and using log.error() to print reason why it is empty.

Reply:
in this plugin we don't have yet container image, so the reason why plugin failed is because there aren't any srpms,
also in error message explaining that it would produce empty image

Reply:
I was worrying about a developer seeing this a while from now and not getting the full picture of why not proceeding with an empty `urls` list, or what and empty `urls` list would mean. It would also be nice to have the same message in the error log and in the exception since (from my experience) OSBS users may read just the exception message (due to the size of our logs) before opening bugs.

This LGTM and I will leave the "having the same message in both the error log and the exception"  as a (non-blocker) nitpick

Reply:
added same msg to both